docs/Porting_Guide.md

```diff
@@ -1,16 +1,28 @@
 # Porting Guide
 
 ## Contents
-> [Policy & Causion](policy--causion)  
+> [Policy & Caution](policy--caution)  
 > [Kernel & System](#kernel--system)  
 > [File System](#file-system)  
 > [Network](#network)  
 > [Audio](#audio)  
 > [Confirmation of Porting](#confirmation-of-porting)
 
-## Policy & Causion
-1. Use driver, framework and system call APIs.  
+## Policy & Caution
+1. Driver Implementation Recommendation
+Instead of hooking the default driver interface code (which is a part of SoC package) to the platform,
+it is advisable to include the drivers according the TizenRT platform driver model.
```

Comment:
You didn't add two spaces at end of previous line. What is intention? Do you want to write them in same line or different lines?